include/hamlib/rig.h

```diff
@@ -111,6 +112,7 @@
     t.tv_nsec = 0;\
     nanosleep(&t,NULL);\
   } while(0)
+#endif
```

Comment:
This change may not be needed with the most recent updates to master as Mike pulled all of this out of rig.h into a separate file in src.